istio/istio/base/istio-rules.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
Since we have migrated to manifest v3, for most of the cases, we won't suggest adding files into base folder but base_v3 folder. 

Because we kind-of "freeze" all components' base directory and treat them as read-only.

It's okay if istio folder OWNERS think we can make the change.

Reply:
@maganaluis can you please accommodate this change and instead make this in base_v3?

Reply:
Removed any references on the base folder.